pyproject.toml

```diff
@@ -48,6 +48,7 @@ python-dotenv = "*"
 python-magic = "*"
 rfc3339-validator = "*"
 sentry-sdk = {extras = ["flask"], version = "*"}
+setuptools= "77.0.3"
```

Comment:
Had to pin to[ fix issue](https://github.com/pypa/setuptools/pull/4911)

Reply:
Please include an explanation as a comment.